### ISSUE TITLE
Switch glyph at cursor short-cut

### ISSUE THREAD

**Issue by tderkert** [TypeResources/typeresources.github.io#2]
Similar to how you in Glyphs can press **Cmd+Shift+Up/Down** to switch a glyph at cursor to next/previous glyph. [Mouse Trap](https://craig.is/killing/mice) could be a good tool to implement the key press controls. An array of glyphs would be needed and a decision on which glyphs to include.

**Comment by larsenwork** [comment]
sounds a bit advanced but could work if there were some unicode value object we could use.

**Comment by tderkert** [comment]
Yeah, or we just keep a long ordered array of all glyphs, find glyph in array and switch it for next or previous index. Not beautiful but could work for all "regular" alphabet characters. Not sure

**Comment by larsenwork** [comment]
Having a manual array seems a bit too limited though. For it to work you should be able to jump also from e.g. `>` and `Ѿ` to the corresponding next character so we'd have to create that array from some unicode list somewhere

**Comment by tderkert** [comment]
You have a point. I agree that ideally it should support as many glyphs as possible. Maybe too much work for too little use case, for now. More of a neat nice to have in the future.

**Comment by tderkert** [comment]
Some good list of glyphs: https://github.com/google/fonts/tree/master/tools/encodings
